lib/bedrock.js

```diff
@@ -368,6 +368,37 @@ function _setupUncaughtExceptionHandler(logger, logPrefix) {
   }
 }
 
+function _setupSignalHandler(logger, logPrefix) {
+  const SIGNALS = {
+    /*
+      The SIGHUP (“hang-up”) signal is used to report that the
```

Comment:
Use regular double quote char here and elsewhere.

```suggestion
      The SIGHUP ("hang-up") signal is used to report that the
```